lms/djangoapps/ccx/utils.py

```diff
@@ -127,6 +127,31 @@ def get_ccx_for_coach(course, coach):
     return None
 
 
+def get_ccx_by_ccx_id(course, coach, ccx_id):
+    """
+    Looks to see if user is coach of a CCX for this course.  Returns the CCX or
+    None.
```

Comment:
@nedbat yes, now a course can have multiple ccx, This method returns a specific ccx which i am using to verify that this coach on this course has access on this ccx.